profiler/src/ProfilerEngine/Datadog.Profiler.Native.Windows/WindowsThreadsCpuManager.cpp

```diff
@@ -94,8 +94,16 @@ void ThreadsCpuManager::LogCpuTimes()
                 builder << std::setw(6) << threadEntry.th32ThreadID << " | ";
                 builder << std::setw(8) << ThreadMs << " ms [";
                 builder << std::setw(5) << std::setprecision(2) << percent << " %]  ";
-                auto name = *element->second->GetName();
-                builder << shared::ToString(name);
+                auto name = element->second->GetName();
```

Comment:
nit: can you use `auto*` . Just to ease the reading.
Thx :)

Reply:
✅ done